track_and_trace/server/system/submit_utils.js

```diff
@@ -49,48 +56,75 @@ const awaitServerPubkey = () => {
 }
 
 const encodeHeader = (signerPublicKey, batcherPublicKey, payload) => {
+  const GRID_NAMESPACE = hash(FAMILY_NAME,6) + '01'
```

Comment:
This would be the schema namespace, since the Grid namespace is just `621dee`

Reply:
updated